.github/workflows/ci.yml

```diff
@@ -18,6 +18,11 @@ jobs:
     uses: ./.github/workflows/reusable-verification.yml
     secrets: inherit
 
+  unit-test-js:
```

Comment:
Referring to my other comment, this can be removed, in case you agree with me. 😁